src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py

```diff
@@ -363,7 +363,8 @@ def carp_master(self, fobj, ifname, vhid, event, user_override, forcetakeover):
                 # For reboots, /tmp is cleared by virtue of being a memory device.
                 # If someone does a kill -9 on the script while it's running the lockfile
                 # will get left dangling.
-                self.logger.warn('Aquired failover master lock')
+                self.middleware.call_sync('failover.call_remote', ['failover.sync_keys_with_remote_node'])
```

Comment:
We dont need this here, I misunderstood when you asked.

We will already sync when entering in backup and thats enough